Pocket/pocket-ios
# Add PocketShare and handle related share urls (inbound and outbound)

## Summary
* This PR removes previous calls to `shortUrl` and sends `PocketShare.shareUrl` when sharing. It also processes `pocket.co/share/[slug]` and `pocket.co/read/[slug]` inbound urls.

## References 
* Branch name

## Implementation Details
* Add `CreateShareLink` mutation
* Att `ShareSlug` query
* Remove calls to `shareUrl` and related fixtures
* Add `PocketShareRoute` and `PocketReadRoute` to handle new `URL` patterns
* update `MainViewModel`, add `pocketShareUrlRoutingAction` to handle new urls

## Test Steps
* Build run, share some items and make sure they are in the new format
* Obtain new format urls, and make sure they are handled in the app as expected (e.g. if you have the item saved, it should show in the Reader)

## PR Checklist:

## Screenshots
